components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -557,6 +589,23 @@ void RewardsServiceImpl::CreateWallet(CreateWalletCallback callback) {
   }
 }
 
+#if defined(OS_ANDROID)
+void RewardsServiceImpl::CreateWalletAttestationResult(
+    CreateWalletCallback callback,
+    bool result,
+    const std::string& result_string) {
+  if (result) {
+    auto on_create = base::BindOnce(
```

Comment:
we shouldn't be duplicating this code here, you can pass the original `on_create` callback from https://github.com/brave/brave-core/pull/3499/files#diff-a5d16be9c9b52a6df7b866b1d75d3f5fR569. Only `bat_ledger_->CreateWallet("", std::move(on_create));` should be in the `!defined(OS_ANDROID)` block

Reply:
Fixed in https://github.com/brave/brave-core/commit/7c2d934860bd67d6d4f03c3658e5af1ddedeeaca